applications/vessel-history/src/features/profile/components/InfoFieldHistory.tsx

```diff
@@ -24,13 +25,17 @@ const InfoFieldHistory: React.FC<ListItemProps> = ({
 }): React.ReactElement => {
   const { t } = useTranslation()
 
-  const formatedDate = useCallback((date: string | null = null) => {
-    return date
-      ? [DateTime.fromISO(date, { zone: 'UTC' }).toLocaleString(DateTime.DATETIME_MED), 'UTC'].join(
-          ' '
-        )
-      : ''
-  }, [])
+  const defaultTitle = useMemo(() => {
+    return `${label} History for ${vesselName}`
```

Comment:
translate this?

Reply:
It's the default value fallback when no translation is found.